input/vehicle-management-services.yaml

```
@@ -760,6 +760,58 @@ paths:
             application/json:
               schema:
                 $ref: "#/components/schemas/Error"
+
+  /v1/trucks/{truckId}/temperature-readings:
```

We have other endpoints with camelcase. Also should the entity name be just "temperature" (e.g. you ask "what's the temperature" instead of  "what's the temperature value") 🤔 This would then change this endpoint to "temperatures".
```suggestion
  /v1/trucks/{truckId}/temperatures:
```